ceph/cephfs/README.md

```diff
@@ -16,16 +16,16 @@ docker build -t cephfs-provisioner .
 * Create a Ceph admin secret
 
 ```bash
-ceph auth get client.admin 2>&1 |grep "key = " |a^C '{print  $3'} |xargs echo -n > /tmp/secret
+ceph auth get client.admin 2>&1 |grep "key = " |awk '{print  $3'} |xargs echo -n > /tmp/secret
 kubectl create secret generic ceph-secret-admin --from-file=/tmp/secret --namespace=kube-system
 ```
 
 * Start CephFS provisioner
 
-Assume kubeconfig is at `/root/.kube`:
+The following example uses `cephfs-provisioner-1` as the identity for the instance and assumes kubeconfig is at `/root/.kube`. If there are multiple provisioners, each should have a different identity.
```

Comment:
e.g. the efs one is stateless because any "state" is based on what is on the efs share